component/Products/Product.css

```diff
@@ -0,0 +1,30 @@
+.filter_and_Clear{
+    justify-content: space-between;
+}
+
+.range_input{
+    width: 100%;
+}
+.ranges{
+    justify-content: space-between;
+}
+.product_category{
+    flex-direction: column;
+}
+.product_rating{
+    flex-direction: column;
+}
+.sidebar{
```

Comment:
You can leave a space between the selector and opening curly bracket and leave a line after the closing curly bracket, it helps in readability.